src/components/EmptyState/EmptyStatePattern.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
```

Comment:
can we move this to `EmptyState/Patterns/EmptyStatePattern` etc. to mimic #268 ?